### PR TITLE
Support relative HTTP redirects

### PR DESCRIPTION
**Pull request by theli-ua**

### CODE REVIEW

**Comment by theli-ua** [comment]
Test url: _http://zv.fm/download/2190677

File: src/libtomahawk/utils/NetworkReply.cpp [hunk 1]
```diff
@@ -90,7 +90,14 @@ NetworkReply::metaDataChanged()
             else
             {
                 disconnectReplySignals();
-                load( redir.toUrl() );
+                if(redir.toUrl().isRelative())
```
Comment (muesli) [inline src/libtomahawk/utils/NetworkReply.cpp]: Style: if ( redir.toUrl().isRelative() )

Wanna commit that quickly and I'll merge? ;-)

**Comment by theli-ua** [comment]
fixed to match your style

**Comment by muesli** [comment]
<3

**Pull request merged**